template projects/chatgpt/e3e144d2-7aab-423b-957e-7f95646ff11f.json

```diff
@@ -0,0 +1,165 @@
+{
+  "attributes": {
+    "root": true,
+    "template": true
+  },
+  "description": "Build a basic prompt and ask ChatGPT.\n\nOpenAI api key is required.\n\nMore: https://openai.com/api/",
+  "interface": {
+    "inputs": {},
+    "outputs": {}
+  },
+  "implementations": {
+    "cranq": {
+      "nodes": {
+        "8253ddd3-0709-42eb-b91e-309f5d3acf43": {
+          "coordinates": [
+            -672,
+            13
+          ],
+          "description": "",
+          "params": {},
+          "prototype": "d227e22b-e827-4c9d-99db-f96da179bd52",
+          "name": "start"
+        },
+        "79688e50-8550-40b8-afea-3c7497fcb9f3": {
+          "coordinates": [
+            253,
+            42
+          ],
+          "description": "",
+          "params": {},
+          "prototype": "1d2a6438-e4bd-4105-b491-cebbb4cae348",
+          "name": "log"
+        },
+        "ffaff8ea-0269-480d-8240-6e687479963f": {
+          "coordinates": [
+            -68.19999999999999,
+            -55.615624999999994
+          ],
+          "name": "ask ChatGPT",
+          "prototype": "d4ebabb4-abcf-4ee4-9598-c37eba05e733",
+          "params": {}
+        },
+        "1485f5e9-a533-4137-b6c8-79fcd9eb2fea": {
+          "coordinates": [
+            66.80000000000001,
+            -200.5375
+          ],
+          "name": "get answer",
+          "prototype": "d8608f30-9132-4d0f-992e-9d66f6636256",
+          "params": {
+            "927e6b17-97e0-44de-887f-57204b7a1f9b": "[\"choices\", 0, \"text\"]"
+          }
+        },
+        "c216dc40-6ec2-48d5-9975-4f5e70c90179": {
+          "coordinates": [
+            -518,
+            -116.5
+          ],
+          "name": "question -  FILL IT!!!",
+          "prototype": "485b6519-9402-4e78-8599-47ad52c92b2f",
+          "params": {
+            "543f8ad2-7755-44f6-bc9b-e6208c87b15a": "\"\""
+          },
+          "description": "Fill the value of the data port with your question."
+        },
+        "d39603a2-2007-4aff-ab76-0895fcc48f12": {
+          "coordinates": [
+            -459,
+            92.5
+          ],
+          "name": "ChatGPT params - FILL IT!!!",
+          "prototype": "485b6519-9402-4e78-8599-47ad52c92b2f",
+          "params": {
+            "543f8ad2-7755-44f6-bc9b-e6208c87b15a": "{\"apiKey\": \"\"}"
```

Comment:
I think we should not provide any parameter here:
- port gets into warning state without the parameter that may trigger the user some action
- if the user runs the project without filling in the parameters the process clearly stops at the store making the issue obvious.